examples/combobox/js/listboxOption.js

```diff
@@ -2,7 +2,7 @@
 *   This content is licensed according to the W3C Software License at
 *   https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
 */
-var ListboxOption = function (domNode, listboxObj) {
+var Option = function (domNode, listboxObj) {
```

Comment:
`Option` is a standard constructor.

https://html.spec.whatwg.org/multipage/form-elements.html#the-option-element

This would override the standard constructor, which works fine so long as there isn't other code that uses it.

Until we rewrite examples to use modules, at least, we should avoid global variables with the same names as standard constructors.

Reply:
I noticed now that this also fails in the linter:
```
/home/travis/build/w3c/aria-practices/examples/combobox/js/listboxOption.js
  5:5  error  'Option' is already defined as a built-in global variable  no-redeclare
```

Reply:
Fixed in https://github.com/w3c/aria-practices/pull/1255/commits/29e94eee65a532be3dcbbf0e4c9bad5083584454